lib/controller/handlers/callback.js

```diff
@@ -46,9 +46,8 @@ module.exports = {
         } = request
 
         const loginResult = await login(request, user.nameID, user)
-        const isObject = a => a instanceof Object
 
-        if (isObject(loginResult) && !loginResult.success) {
+        if (loginResult?.success === false) {
```

Comment:
Optional chaining (`?.`) is only available since Node 14:
<img width="797" alt="image" src="https://github.com/toriihq/hapi-saml2/assets/170214/6dc622ce-93da-43a1-a3f9-aa76c7855346">

Let's do the old `loginResult && loginResult.success === false` so we will not have to declare this library only supports Node v14 (it currently supports any version of Node).